src/App.jsx

```diff
@@ -22,6 +23,7 @@ function App() {
           <img src={reactLogo} className="logo react" alt="React logo" />
         </a>
       </div>
+      <FavoriteStay />
```

Comment:
This component can be added the product page itself as it is already implemented.

Reply:
thanks, changed that